std/algorithm/iteration.d

```diff
@@ -4289,3 +4304,625 @@ unittest
          [1, 2, 0],
          [2, 1, 0]]));
 }
+
+/**
+Computes the $(WEB https://en.wikipedia.org/wiki/Binomial_coefficient, binomial coefficient)
+of $(D n,k).
+It is also known as "n choose k " or more formally as $(D n!/k!(n-k)).
+
+Params:
+    n = non negative integer
+    k = non negative integer, k <= n
```

Comment:
Yeah, I think either way is fine, whichever you like better.

It would be neat to have some reflection of input contracts in documentation, but input contracts are just arbitrarily complex code snippets, and even in simple cases there's syntactic baggage:
```d
assert(k <= n);
```
In this simple case the `assert(…);` would be noisy if reflected verbatim in generated documentation.